src/common/WalletBalances.vue

```diff
@@ -104,19 +107,21 @@ const onSelectBtnClick = (balanceIdx: number) => {
   isDropMenuShown.value = false
 }
 
-const updateBalances = async (): Promise<void> => {
+const updateBalances = async () => {
   if (!web3ProvidersStore.provider.selectedAddress)
     throw new Error('user address unavailable')
 
-  const address = web3ProvidersStore.provider.selectedAddress
-
   const [stEthValue, morValue] = await Promise.all([
-    web3ProvidersStore.stEthContract.providerBased.value.balanceOf(address),
-    web3ProvidersStore.morContract.providerBased.value.balanceOf(address),
+    web3ProvidersStore.depositContract.providerBased.value.balanceOf(
+      web3ProvidersStore.provider.selectedAddress,
+    ),
+    web3ProvidersStore.rewardsContract.providerBased.value.balanceOf(
+      web3ProvidersStore.provider.selectedAddress,
+    ),
```

Comment:
I think in order to improve readability  you can extract contracts interfaces at the top of the component
```ts
const [depositContract, rewardsContract] = computed(() => [
    web3ProvidersStore.depositContract.providerBased, 
    web3ProvidersStore.rewardsContract.providerBased
    ]
 )
```

and reduce usage calls to
```ts
const [stEthValue, morValue] = await Promise.all([
    depositContract.balanceOf(),
    rewardsContract.balanceOf(),
])
```

Reply:
I can't do decomposition like 

```
const [depositContract, rewardsContract] = computed(() => [
    web3ProvidersStore.depositContract.providerBased, 
    web3ProvidersStore.rewardsContract.providerBased
    ]
 )
```

Computed returns me a `Proxy` object witch `value` field. And it won't decrease count of code lines